src/components/CardList/CardList.js

```diff
@@ -1,19 +1,23 @@
 import Card from '../Card/Card';
 import { CardGrid } from '../UI/CardGrid.styled';
+import useStore from '../useStore';
 
-export default function CardList({ entries }) {
+export default function CardList({ onDeleteEntry }) {
+	const entries = useStore(state => state.entries);
 	return (
 		<CardGrid>
 			{entries.map(entry => (
 				<Card
 					key={entry.id}
+					id={entry.id}
 					category={entry.category}
 					name={entry.name}
 					address={entry.address}
 					products={entry.products}
 					information={entry.information}
 					visited={entry.visited}
 					rating={entry.rating}
+					onDeleteEntry={onDeleteEntry}
```

Comment:
Not quite sure if this line does anything. Since you do not use this prop in the card-component.

Reply:
You are right, I don't need this anymore when using zustand.